lib/web_ui/pubspec.yaml

```diff
@@ -5,17 +5,17 @@ environment:
   sdk: ">=2.10.0-0 <3.0.0"
 
 dependencies:
-  meta: 1.1.8
+  js: 0.6.3-nullsafety.3
+  meta: 1.3.0-nullsafety.6
 
 dev_dependencies:
   analyzer: 0.39.15
   archive: 2.0.13
   http: 0.12.1
   image: 2.1.13
-  js: 0.6.1+1
   mockito: 4.1.1
   path: 1.8.0-nullsafety.3
-  test: 1.14.3
+  test: 1.16.0-nullsafety.9
   quiver: 2.1.3
```

Comment:
fwiw quiver does now have a null safety release, if you want to use it

Reply:
Done.